src/MountedTraversal.js

```diff
@@ -50,6 +50,9 @@ export function instHasClassName(inst, className) {
     return false;
   }
   let classes = findDOMNode(inst).className || '';
+  if (typeof classes !== 'string') {
+    return false;
```

Comment:
That snippet above should introduce support for SVGs in all cases as well. We could also invert the `if` block if we wanted to be able to cleanly delete the `className` code in the future.